origocli/commands/events.py

```diff
@@ -12,16 +13,29 @@ class EventsCommand(BaseCommand):
     __doc__ = f"""Oslo :: Events
 
 Usage:
-  origo events put <datasetid> <versionid> [--file=<file> options]
-  origo events stat <datasetid> [options]
+  origo events cat <dataset-uri> [options]
+  origo events create-stream <dataset-uri> [--skip-raw] [options]
+  origo events delete-stream <dataset-uri> [options]
+  origo events enable-subscription <dataset-uri> [options]
+  origo events disable-subscription <dataset-uri> [options]
+  origo events enable-sink <dataset-uri> --sink-type=<sink_type> [options]
+  origo events disable-sink <dataset-uri> --sink-id=<sink_id> [options]
+  origo events put <dataset-uri> [--file=<file> options]
+  origo events stat <dataset-uri> [options]
 
 Examples:
-  echo '{{"hello": "world"}}' | origo events put test-event 1
-  echo '[{{"hello": "world"}}, {{"world": "hello"}}]' | origo events put test-event 1
-  cat /tmp/event.json | origo events put test-event 1
-  origo events put test-event 1 --file=/tmp/event.json
-  origo events stat test-event
-  origo events stat test-event --format=json | jq ".last_hour.events"
+  origo events cat ds:my-dataset-id/1
+  origo events cat my-dataset-id/1
+  origo events cat my-dataset-id
+  origo events create-stream ds:my-dataset-id/1
+  origo events enable-sink ds:my-dataset-id/1 --sink-type=s3
+  origo events disable-sink ds:my-dataset-id/1 --sink-id=ab12c
+  echo '{{"hello": "world"}}' | origo events put ds:my-dataset-id/1
+  echo '[{{"hello": "world"}}, {{"world": "hello"}}]' | origo events put ds:my-dataset-id/1
+  cat /tmp/event.json | origo events put ds:my-dataset-id/1
+  origo events put ds:my-dataset-id/1 --file=/tmp/event.json
```

Comment:
Bør vi også tillate `origo events put ds:my-dataset-id/1 --data '{{"hello": "world"}}'` ?

Reply:
Ja, kan legge til en `--data=<payload>`. Dette er kanskje like aktuelt for andre kommandoer som aksepterer data via stdin eller `--file`? (gjør det i så fall tillatt for `put` spesifikt nå)